modules/quant_purger/config/install/quant_purger.settings.yml

```diff
@@ -16,3 +16,7 @@ tag_blocklist:
   - 'local_task'
 path_blocklist:
   - '/admin/*'
+tag_allowlist:
+  - ''
```

Comment:
The reason I did it this way was because I added the form fields and saved the form (without adding anything) and looked at the `sites/default/sync/quant_purger.settings.yml` file and that's the format it showed. So, I think it should be okay.